### DISCUSSION TITLE
Jetson Ideal Temperature

### DISCUSSION THREAD

**Discussion by KCHJ1999** [prl-mushr/mushr#71] [General]
![WhatsApp Image 2022-02-06 at 5 07 46 PM](https://user-images.githubusercontent.com/96906767/152674375-fefddc3f-c7db-434f-9b69-b403e4b06bfc.jpeg)

Is this temperaure okay for the jetson while the car is on? Will this temperature cause my components to overheat and may i know the temperature which is ideal for jetson and which temperature is over the threshold? Thank you.

**Comment by palanc** [comment]
Hi @KCHJ1999 - please see this post on NVIDIA's forum: [https://forums.developer.nvidia.com/t/temperatures-for-jetson-nano/181675](https://forums.developer.nvidia.com/t/temperatures-for-jetson-nano/181675)

As well as page 4 of this document that the above post references (you'll need an NVIDIA account to view it): [https://developer.nvidia.com/embedded/dlc/jetson-nano-thermal-design-guide-1.6](https://developer.nvidia.com/embedded/dlc/jetson-nano-thermal-design-guide-1.6) 

In short, since the maximum operating temperatures of the CPU and GPU are 97.5 degrees celsius, the temperatures that you're seeing should be fine.
